sql/src/test/groovy/io/crate/integrationtests/TableFunctionIt.groovy

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests
+
+import org.junit.Test
+
+import static io.crate.testing.TestingHelpers.printedTable
+
+class TableFunctionIt extends SQLTransportIntegrationTest {
+
+    @Test
+    public void testSelectFromUnnest() throws Exception {
+        execute("select * from unnest([1, 2], ['Trillian', 'Marvin'])")
+        assert response.rowCount() == 2L
+        assert printedTable(response.rows()) == "" +
+            "1| Trillian\n" +
+            "2| Marvin\n"
+    }
+
+    @Test
+    public void testSelectFromUnnestWithOrderByAndLimit() throws Exception {
+        execute("select * from unnest([1, 2], ['Trillian', 'Marvin']) order by col0 limit 1")
```

Comment:
hm not sure if this really tests the ordering. afaik the output is the same without order by..